ca_gazebo/models/create_april_tags/model.sdf

```diff
@@ -0,0 +1,92 @@
+<?xml version='1.0'?>
+<sdf version='1.6'>
+  <model name='create_april_tags'>
+    <static>1</static>
+    
+    <link name='AprilTag1'>
+      <pose>4 4 0 0 0 0</pose>
+      <self_collide>0</self_collide>
+      <kinematic>0</kinematic>
+      <visual name='AprilTag1_Visual'>
+        <geometry>
+          <box>
+            <size>0.5 0.5 0.05 </size>
+          </box>
+        </geometry>
+        <material>
+          <script>
+            <uri>model://create_april_tags/materials/scripts </uri>
+            <uri>model://create_april_tags/materials/textures </uri>
+            <name>AprilTag1</name>
+          </script>
+        </material>
+        <cast_shadows>1</cast_shadows>
+        <transparency>0</transparency>
+      </visual>
+    </link>
+
+    <link name='AprilTag2'>
+      <pose>4 -4 0 0 0 0</pose>
+      <self_collide>0</self_collide>
+      <kinematic>0</kinematic>
+      <visual name='AprilTag2_Visual'>
+        <geometry>
+          <box>
+            <size>0.5 0.5 0.05 </size>
+          </box>
+        </geometry>
+        <material>
+          <script>
+            <uri>model://create_april_tags/materials/scripts/create_april_tags.material </uri>
+            <name>AprilTag2</name>
+          </script>
+        </material>
+        <cast_shadows>1</cast_shadows>
+        <transparency>0</transparency>
+      </visual>
+    </link>
+
+    <link name='AprilTag3'>
+      <pose>-4 4 0 0 0 0</pose>
+      <self_collide>0</self_collide>
+      <kinematic>0</kinematic>
+      <visual name='AprilTag3_Visual'>
+        <geometry>
+          <box>
+            <size>0.5 0.5 0.05 </size>
+          </box>
+        </geometry>
+        <material>
+          <script>
+            <uri>model://create_april_tags/materials/scripts/create_april_tags.material </uri>
+            <name>AprilTag3</name>
+          </script>
+        </material>
+        <cast_shadows>1</cast_shadows>
+        <transparency>0</transparency>
+      </visual>
+    </link>
+
+    <link name='AprilTag4'>
```

Comment:
Oh, you have all the aprilTags in the same model. Check how Steven did it. That's a cleaner approach. Nevertheless, keep yours.